pages/views/_layouts/landing.njk

```diff
@@ -15,6 +15,8 @@
       {% endif %}
       {{ content | safe or 'No content.' }}
     </div>
+    {% from 'benefit.njk' import benefit with context %}
+    {{ benefit ( site.benefit ) }}
```

Comment:
mb can we immediately use {% for %} here (whole file benefit.njk)? without macro